src/markdoc/partials/quickstarts/golem-in-a-browser.md

```diff
@@ -2,12 +2,12 @@
 
 In most of our examples, we demonstrate how to run a requestor script in Node.js. However, you can also run your scripts in a browser context. This example will explain how to do it.
 
-Before getting started, you need to install and launch the Yagna service in version 0.13.0 or later. It can be installed using instructions for manual Yagna installation available [here](/docs/creators/javascript/examples/tools/yagna-installation-for-requestors).
-
-{% /alert %}
+Before getting started, you need to install and launch the Yagna service in version 0.15.0 or later. It can be installed using instructions for manual Yagna installation available [here](/docs/creators/javascript/examples/tools/yagna-installation-for-requestors).
```

Comment:
```suggestion
Before getting started, you need to install and launch the Yagna service in version 0.15.2 or later. It can be installed using instructions for manual Yagna installation available [here](/docs/creators/javascript/examples/tools/yagna-installation-for-requestors).
```